test/Microsoft.AspNet.Mvc.Extensions.Test/ViewComponents/JsonViewComponentResultTest.cs

```diff
@@ -37,9 +40,27 @@ public void Execute_UsesFormatter_WithSpecifiedSerializerSettings()
             Assert.Equal("{\r\n  \"foo\": \"abcd\"\r\n}", Encoding.UTF8.GetString(buffer.ToArray()));
         }
 
+        [Fact]
+        public void Execute_UsesDefaultSerializerSettings_IfNotProvided()
```

Comment:
It might be better to way `..._UsesSerializerSettingsFromOptions_...` - `Default` kinda fits the old semantic.